config/gta3/cleo.xml

```diff
@@ -0,0 +1,958 @@
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb00" Name="DELETE_FILE" Hash="0xdd49d9c1">
+      <Args>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb01" Name="DELETE_DIRECTORY" Hash="0xd48a399c">
+      <Args>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+        <Arg Type="INT"/>
+      </Args>
+    </Command>
+    <Command ID="0xb02" Name="MOVE_FILE" Hash="0xcb9d1de4">
+      <Args>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb03" Name="MOVE_DIRECTORY" Hash="0xa3360aff">
+      <Args>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb04" Name="COPY_FILE" Hash="0xf29cb2cc">
+      <Args>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb05" Name="COPY_DIRECTORY" Hash="0x79b66ee9">
+      <Args>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+        <Arg Type="STRING" AllowPointer="true" PreserveCase="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb10" Name="BITWISE_AND" Hash="0x5e21b170">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb11" Name="BITWISE_OR" Hash="0x7e03c9ad">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb12" Name="BITWISE_XOR" Hash="0xe112b461">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb13" Name="BITWISE_NOT" Hash="0xc315e20e">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb14" Name="MOD" Hash="0xb40a4474">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb15" Name="BITWISE_SHR" Hash="0x12041cc6">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb16" Name="BITWISE_SHL" Hash="0xc3007ec0">
+      <Args>
+        <Arg Type="INT"/>
+        <Arg Type="INT"/>
+        <Arg Type="INT" Out="true"/>
+      </Args>
+    </Command>
+    <Command ID="0xb17" Name="BITWISE_AND_INT_VAR_BY_VAL" Hash="0xe0395187">
```

Comment:
Name doesn't seem R*-wise, I don't know either which would be the correct name since there never was an alternator with an single command able to do operations on ANY VAR and ANY VALUE/VAR.

Duplicate the command for each value type? Does this needs an alternator?